javalib/src/main/scala/java/net/PlainSocketImpl.scala

```diff
@@ -0,0 +1,340 @@
+package java.net
+
+import scala.scalanative.native._
+import scala.scalanative.runtime.ByteArray
+import scala.scalanative.posix.errno._
+import scala.scalanative.posix.sys.socket
+import scala.scalanative.posix.sys.socketOps._
+import scala.scalanative.posix.netinet.in
+import scala.scalanative.posix.netinet.inOps._
+import scala.scalanative.posix.netinet.tcp
+import scala.scalanative.posix.arpa.inet
+import scala.scalanative.posix.netdb._
+import scala.scalanative.posix.netdbOps._
+import scala.scalanative.posix.sys.ioctl._
+import scala.scalanative.posix.fcntl._
+import scala.scalanative.posix.sys.select._
+import scala.scalanative.posix.sys.selectOps._
+import scala.scalanative.posix.unistd.{close => cClose}
+import java.io.{FileDescriptor, IOException, OutputStream, InputStream}
+
+private[net] class PlainSocketImpl extends SocketImpl {
```

Comment:
Not really, I may have been inspired by Harmony but the implementation is different in most cases. Should I mention it here?